This patch updates sam_record.hpp to include the cstdint package. The package is missing in the unmodified source, which stops the program from compiling.
Author: David Schmidt
diff --git a/src/smithlab_cpp/sam_record.hpp b/src/smithlab_cpp/sam_record.hpp
index f444565..3b0ef80 100644
--- a/src/smithlab_cpp/sam_record.hpp
+++ b/src/smithlab_cpp/sam_record.hpp
@@ -19,6 +19,7 @@
 #ifndef SAM_RECORD_HPP
 #define SAM_RECORD_HPP
 
+#include <cstdint>
 #include <string>
 #include <vector>
 #include <iostream>
